tremor-script/src/grammar.lalrpop

```diff
@@ -23,7 +24,172 @@ use beef::Cow;
 
 grammar<'input>;
 
-////////////////////////////// Root statements //////////////////////////////
+////////////////////////////// Troy statements //////////////////////////////
+// These statements for the root of the language.
+
+/// Root
+pub(crate) Deploy : DeployRaw<'input> = {
+    <config:ConfigDirectives> <stmts:DeployStmts> "<end-of-stream>"? => DeployRaw{config, stmts},
+    <stmts:DeployStmts> "<end-of-stream>"? => DeployRaw{config: vec![], stmts},
+}
+
+DeployStmts: DeployStmtsRaw<'input> = {
+    <stmt:DeployStmt> ";" <list:DeployStmts> => {
+      let mut list = list;
+      list.insert(0,stmt);
+      list
+  },
+  <item:DeployStmt> ";"? => vec![item],
+}
+
+DeployStmt: DeployStmtRaw<'input> = {
```

Comment:
same for the defines, Yatin spotted that missing in trickle and is fixing it as part of the subqueries, we should do it here as well

Reply:
All top level troy statements now support documentation comments and 🚀 to @yatinmaan for spotting our omission and mistake! Thanks Yatin!